code/modules/projectiles/gun.dm

```diff
@@ -196,9 +196,9 @@
 
 /obj/item/gun/afterattack_secondary(mob/living/victim, mob/living/user, params)
 	if(!isliving(victim) || !IN_GIVEN_RANGE(user, victim, GUNPOINT_SHOOTER_STRAY_RANGE))
-		return ..() //if they're out of range, just shootem.
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
 	if(!can_hold_up)
-		return ..()
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
```

Comment:
This isn't documented in your PR. Why are you changing this behavior?

Reply:
My bad due to late coding. I'll update it once it'll be ready for review. It's done to prevent accidental shooting when you are just out of range

Reply:
Alright

Reply:
That is not only a horrible keybind (unintuitive + not all mice have middle mouse button) that also conflicts with pointing.

Reply:
I mean, it is an agressive/hostile pointing in combat mode, and a regular pointing in non-combat mode.

What are the other ways of fixing M90 issue with gunpointing? A new hotkey?

Reply:
Probably rebinding gunpoint to ctrl-click or something, I thought that's already what it was. Right-click was never a good idea for it